packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,659 @@
+        ],
+        "properties": {
+          "srcAccountId": {
+            "type": "string"
+          },
+          "destAccountId": {
+            "type": "string"
+          },
+          "assetId": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          },
+          "amount": {
+            "type": "number"
+          }
+        }
+      },
+      "CompareAndSetAccountDetailRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Compare And Set Account Detail.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "key",
+          "value"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "key": {
+            "type": "string"
+          },
+          "value": {
+            "type": "string"
+          },
+          "oldValue": {
+            "type": "string"
+          },
+          "check_empty": {
+            "type": "boolean"
+          }
+        }
+      },
+      "GetAccountRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account.",
+        "type": "object",
+        "required": [
+          "accountId"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetBlockRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Block.",
+        "type": "object",
+        "required": [
+          "height"
+        ],
+        "properties": {
+          "height": {
+            "type": "integer"
+          }
+        }
+      },
+      "GetSignatoriesRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Signatories.",
+        "type": "object",
+        "required": [
+          "accountId"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Transactions.",
+        "type": "object",
+        "required": [
+          "txHashesList"
+        ],
+        "properties": {
+          "txHashesList": {
             "type": "array",
-            "items": {}
+            "items":{
+              "type": "string"
+            }
+          }
+        }
+      },
+      "GetPendingTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Pending Transactions.",
+        "type": "object",
+        "required": [
+          "pageSize"
+        ],
+        "properties": {
+          "pageSize": {
+            "type": "integer"
+          },
+          "firstTxHash": {
+            "type": "string"
+          }
```

Comment:
I suggest to add also optional params
- `first_tx_time`
- `last_tx_time`
In [documentation](https://iroha.readthedocs.io/en/develop/develop/api/queries.html#result-pagination) there are more optional params - but they are useless in the command (pending transaction is not committed, so it is not in any block, so filtering to specific blocks is not useful)